scripts/deploy.backend.sh

```diff
@@ -20,6 +22,7 @@ case $ENV in
   # URL used by issuer in the issued verifiable credentials (tipically hard-coded)
   # Represents more an ID than a URL
   POUH_ISSUER_VC_URL="https://id.decideai.xyz/"
+  DERIVATION_ORIGIN="https://oisy.com"
```

Comment:
Note: Is there a reason to use a new variable, not e.g. `VITE_OISY_URL=https://oisy.com`  or `VITE_AUTH_DERIVATION_ORIGIN=https://tewsx-xaaaa-aaaad-aadia-cai.icp0.io` (used for II logins)?

If a different variable is used, will it need to be added to 1password & .github/workflows/deploy-to-environment.yml ?

If the new variable is usually the same as say the II derivation origin, would it make sense to default to using that if not specfied?

Reply:
Yes, it should be the same derivation origins as in the frontend. However, in the frontend, the frontend origin might not be set, which reverts to using the window location origin. But in the backend I need this env var always set.

Does the backend have access to the same env vars as the frontend?

Reply:
Normally the backend build doesn't care about the frontend env vars but for sure we can make it care, either with a built in variable or by providing this as as install arg (nicer, as then we still have the same Wasm everywhere).  Do you want help with this or maybe sync with @inc-man who takes care of deployments to staging, beta and soon also production?

Reply:
This PR provides it as install arg, isn't that enough?

Let me sync with @inc-man !